### PR TITLE
Project Review Trials

### PR DESCRIPTION
**Pull request by afikanzoro**
- This is a project from Microverse trials
- It is a site for selling trombones
- It is a simple website that includes links, a body section and a YouTube trombone video

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
 </head>
 <body>
    
-<div id="wrapper">
+<div class="wrapper">
   <header id="header">
     <div class="logo">
```
Comment (afikanzoro) [inline index.html:14]: Please use class instead of id for blocks. It will keep your code more organized

File: index.html [hunk 2]
```diff
@@ -19,7 +19,7 @@
       />
     </div>
 
-    <nav id="navigation-bar">
+    <nav class="navigation-bar">
       <ul>
         <li><a class="nav-link" href="#features">Features</a></li>
```
Comment (afikanzoro) [inline index.html:24]: As mentioned above, please avoid using id on blocks. It should be used on individual elements instead

File: .github/workflows/linters.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
```
Comment (afikanzoro) [inline .github/workflows/linters.yml:13]: The Linter checks are not visible on your repository. Kindly make sure that the Linter checks are visible

**Review by afikanzoro — COMMENTED** [review]
#Changes required :recycle: 

Good job reaching this far. Howerver, kindly follow the suggested changes on your code. Cheers and Happy codding

**Review by andersonlebon — APPROVED** [review]
## Hello @afikanzoro

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 

## Optional suggestions

- [ ] When giving a review consider using emojis or gifs to make your review more human-friendly 👨  👩   and not robot 🤖  👽 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**